### PR TITLE
Create Expenditure class with skeletal functions code

### PR DESCRIPTION
**Pull request by HansHengGit**

### CODE REVIEW

File: src/main/java/seedu/planitarium/ExpenditureList.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package seedu.planitarium;
+
+import java.util.ArrayList;
+
+public class ExpenditureList {
+
+    protected ArrayList<Expenditure> expenditureArrayList;
```
Comment (Teanweijun) [inline src/main/java/seedu/planitarium/ExpenditureList.java:7]: Can be private, ExpenditureList won't see children anyways

File: src/main/java/seedu/planitarium/ExpenditureList.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    public void addExpenditure(String description, double amount) {
+        this.expenditureArrayList.add(new Expenditure(description, amount));
+    }
+
+    public double getExpenditureValue(int index) {
+        double value = 0;
+        try {
+            value =  expenditureArrayList.get(index - 1).amount;
+        } catch (IndexOutOfBoundsException e) {
```
Comment (Teanweijun) [inline src/main/java/seedu/planitarium/ExpenditureList.java:21]: Buggy. You might want to use a parser method instead? Perhaps can throw some exception if it fails
As of now the code will still return 0 like nothing happened even though there is something printed
More similar situations below

File: src/main/java/seedu/planitarium/ExpenditureList.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            totalAmount += item.amount;
+        }
+        return totalAmount;
+    }
+
+
+    public void printExpenditureList() {
+        int list_index = 1;
+        for (Expenditure Item : expenditureArrayList) {
+            System.out.println(list_index++ + ". " + Item);
```
Comment (Teanweijun) [inline src/main/java/seedu/planitarium/ExpenditureList.java:48]: Interesting implementation... Might use that for my own portion

**Review by tjiarong — APPROVED** [review]
Ok after update

**Pull request merged**